specification/cognitiveservices/data-plane/Face/preview/v1.0-preview/Face.json

```diff
@@ -0,0 +1,5548 @@
+        "operationId": "Snapshot_Apply",
+        "parameters": [
+          {
+            "$ref": "#/parameters/snapshotId"
+          },
+          {
+            "name": "body",
+            "description": "Request body for applying a snapshot.",
+            "in": "body",
+            "required": true,
+            "x-ms-client-flatten": true,
+            "schema": {
+              "$ref": "#/definitions/ApplySnapshotRequest"
+            }
+          }
+        ],
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "responses": {
+          "202": {
+            "description": "The snapshot applying task was queued successfully.",
+            "headers": {
+              "Operation-Location": {
+                "description": "Operation location with an operation id used to track the progress of applying the snapshot by OperationStatus - Get.",
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "Error response.",
+            "schema": {
+              "$ref": "#/definitions/APIError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Apply snapshot example": {
+            "$ref": "./examples/ApplySnapshot.json"
+          }
+        }
+      }
+    },
+    "/operations/{operationId}": {
+      "get": {
+        "description": "Get status of a long running operation.",
+        "operationId": "Snapshot_GetOperationStatus",
+        "parameters": [
+          {
+            "$ref": "#/parameters/operationId"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "A successful call returns the operation's status.",
+            "schema": {
+              "$ref": "#/definitions/AsyncStatus"
+            }
+          },
+          "default": {
+            "description": "Error response.",
+            "schema": {
+              "$ref": "#/definitions/APIError"
+            }
+          }
+        },
+        "produces": [
+          "application/json"
+        ],
+        "x-ms-examples": {
+          "Get snapshot operation status example": {
+            "$ref": "./examples/GetSnapshotOperationStatus.json"
+          }
+        }
+      }
+    },
+    "/persons": {
+      "get": {
+        "tags": [
+          "PersonDirectory"
+        ],
+        "summary": "Retrieve list of person information in person directory.",
+        "operationId": "PersonDirectory_GetPersons",
```

Comment:
I'm a bit confused by this operation.  The name and summary implies it returns a collection however the response schema is singular.  Can you please clarify?

Reply:
The response returns an array of the schema item.